lib/feature/contact_support/bloc/contact_support_bloc.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:app/app/service/messenger/messenger.dart';
+import 'package:app/di/di.dart';
+import 'package:app/feature/contact_support/contact_support.dart';
+import 'package:app/generated/generated.dart';
+import 'package:bloc/bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:logging/logging.dart';
+
+part 'contact_support_event.dart';
+part 'contact_support_state.dart';
+part 'contact_support_bloc.freezed.dart';
+
+class ContactSupportBloc
+    extends Bloc<ContactSupportEvent, ContactSupportState> {
+  ContactSupportBloc() : super(const _Initial()) {
```

Comment:
Lets use normal constructors for states/events such as `const ContactSupportState.initial()`